app/css/style.css

```diff
@@ -1,2 +1,92 @@
+* {
```

Comment:
it is common to not commit css files, as they are now auto generated, you can put them in the git ignore